examples/int8_training/peft_adalora_whisper_large_training.py

```diff
@@ -326,11 +331,18 @@ def save_model_hook(models, weights, output_dir):
         weights.pop()
 
 
+def get_base_model(model):
```

Comment:
This is because model becomes a DistributedDataParallel when training in distributed mode. Not sure if this is the correct/intended way to do it, but at least it abstracts it a bit